src/main/java/org/humancellatlas/ingest/archiving/submission/web/ArchiveJobController.java

```diff
@@ -39,13 +40,24 @@ ResponseEntity<?> createArchiveJob(@RequestBody ArchiveJob archiveJob,
         final PersistentEntityResource entityResource = resourceAssembler.toFullResource(persistedArchiveJob);
         return ResponseEntity.created(URI.create(entityResource.getId().getHref()))
                 .contentType(MediaType.APPLICATION_JSON)
-                .body(persistedArchiveJob);
+                .body(entityResource);
     }
 
     private void initResource(ArchiveJob archiveJob) {
         archiveJob.setCreatedDate(Instant.now());
         archiveJob.setOverallStatus(ArchiveJob.ArchiveJobStatus.PENDING);
     }
 
+    @GetMapping("/archiveJobs/{id}")
+    ResponseEntity<?> getArchiveJob(@PathVariable String id,
+                                    PersistentEntityResourceAssembler resourceAssembler) {
+        Optional<ArchiveJob> archiveJob = archiveJobRepository.findById(id);
+
+        if (archiveJob.isEmpty()) {
+            return ResponseEntity.notFound().build();
```

Comment:
Would it make sense to report the id here?

Reply:
@amnonkhen The client already knows the `id`. It seems to me a useless info.